library/src/test/scala/com/chatwork/scala/jwk/ECJWKSpec.scala

```diff
@@ -90,7 +90,7 @@ class ECJWKSpec extends AnyFreeSpec with Matchers with ECJWKJsonImplicits with E
       jwk.keyType shouldBe KeyType.EC
       jwk.curve shouldBe Curve.P_521
       val result = jwk.computeThumbprint.value
-      result.asString shouldBe "EroitZ-og3Ji6ENuMuey6vEz4hA2i56rOJHfrTeDHII"
+      result.asString shouldBe "rz4Ohmpxg-UOWIWqWKHlOe0bHSjNUFlHW5vwG_M7qYg"
```

Comment:
I verified these with https://kjur.github.io/jsrsasign/sample/tool_jwktp.html, but as they're taken from https://jose.ietf.narkive.com/GnnXSWEA/jwk-thumbprint-rfc-7638 and the thumbprints have been checked there, the confidence in them being correct is even higher.